src/sql/parts/query/services/queryEditorService.ts

```diff
@@ -161,7 +166,26 @@ export class QueryEditorService implements IQueryEditorService {
 	public onQueryInputClosed(uri: string): void {
 	}
 
+	private promptFileSavedNotification(savedFilePath: string) {
```

Comment:
I like having the open file option for exported files, but I'm not sure about this part of the change.  If you separate into different PRs I think we can take the changes related to opening exported files per easily.  This change would require more design discussion.